ompy/vector.py

```diff
@@ -424,6 +424,67 @@ def to_keV(self) -> Vector:
         else:
             raise NotImplementedError("Units must be keV or MeV")
 
+    def closest(self, E: ndarray, side: Optional[str]='right',
+                inplace=False) -> Union[Vector, None]:
+        """ Returns a vector with the values are set by
+            E[i] <= E[j] < E[i+1], values[i].
+
+            Args:
+                E: Bin value to find. Value or array.
+                side: 'left': E[i] < E[j] <= E[i+1], 
+                      'right': E[i] <= E[j] <= E[i+1]
+                inplace: Whether to make the change inplace or not.
+            Returns:
+                Vector with the new E axis and the bin content of the bins
+                that contains E.
+        """
+
+        indices = np.searchsorted(self.E, E, side=side)-1
+
+        values = self.values[indices]
+        std = None
+        if self.std is not None:
+            std = self.std[indices]
+        if inplace:
+            self.E = E
+            self.values = values
+            self.std = std
+        else:
+            return Vector(values=values, E=E, std=std, units=self.units)
+
+    def cumulative(self, factor: Optional[Union[float, str]]=None,
+                   inplace=False) -> Union[Vector, None]:
+        """ Calculate the cumulative sum of the vector.
+
+            Args:
+                factor: A factor to multiply to the resulting vector.
+                        Possible values are None, a float or string 'de',
+                        meaning that the factor is determined by the
+                        bin width of the vector.
```

Comment:
Add how the factor is determined? "The factor preserves (...)", or just the formula?

Reply:
Fixed